### DISCUSSION TITLE
Conditional Media new line

### DISCUSSION THREAD

**Discussion by Nome200** [matt8707/hass-config#73] [Q&A]
Hi, how can i get the conditional media to show me the interpret in a different line? When showing title and interpret in one line its always too long.
```
conditional_media:
    template:
      - base
      - base_media
    state_display: >
      [[[
        let media_artist = entity === undefined || entity.attributes.media_artist,
          media_title = entity === undefined || entity.attributes.media_title;
        if (media_title === "Nothing playing" || media_title === "No title" && variables.state === "paused") {
          return "Inget spelas";
        }
        return media_title === "No title" && variables.state === "playing"
          ? "Kein Titel"
          : media_artist === undefined
          ? media_title
          : `${media_title} - ${media_artist}`;
      ]]]
```
Is there a way to have a new line between those?
``` 
: `${media_title} - ${media_artist}`; 
```

**Comment by matt8707** [comment]
Hi, this looks like outdated code, have you tried "Media Marquee" in [2022.3](https://github.com/matt8707/hass-config/releases/tag/2022.3)
If you want a new line you can do something like [this](https://community.home-assistant.io/t/a-different-take-on-designing-a-lovelace-ui/162594/2301)
